### ISSUE TITLE
"Outside block mutiplicator mine" Glitch

### ISSUE THREAD

**Issue by Time0ut64** [seagemgames/tangymines#11]
How to do it:

1. Dig a very large hole.
2. Go to surface and jump on the hole.
3. When ur speed is high enough, go to surface.
4. You will spawn under of the ground and fall. So, be quick on landing...
5. Now every time you mine, it mutipicate on the sides so who is inside on the cave, can be on the mined block. (It's just a ilusion.)
6. You can see the ores more easily and mine them and you can even make plataforms with the multiplicated block to go down safely and if you can go all the way down, you can get some very rare blocks.

In conclusion: this glitch can be very game breaking but, I say to keep the glitch because it's so fun to use it. ;)

![image](https://user-images.githubusercontent.com/70518169/91716566-51376d80-eb66-11ea-834f-47b056b6d4e8.png)

**Comment by saismeesaimonsaimes** [comment]
Breaking out is just a Roblox physics issue. I believe I could fix it quite easily but it'd make mining slightly laggier. The "multiplication" is just the game generating blocks around the newly dug block, which is correct.
